src/input_xml.F90

```diff
@@ -2489,6 +2494,102 @@ subroutine read_tallies_xml()
+                deallocate(t % filters(j) % real_bins)
+                allocate(t % filters(j) % real_bins(Nangle + 1))
+                do iangle = 1, Nangle + 1
+                  t % filters(j) % real_bins(iangle) = -ONE + (iangle - 1) * dangle
+                end do
+                t % filters(j) % real_bins(Nangle + 1) = ONE
+              else
+                call fatal_error("Number of bins for mu filter must be&
+                     & greater than 1 on tally " // trim(to_str(t % id)) // ".")
+              end if
+
+            end if
+
+            ! Set to analog estimator
+            t % estimator = ESTIMATOR_ANALOG
+
+          case ('polar')
+            ! Set type of filter
+            t % filters(j) % type = FILTER_POLAR
+
+            ! Set number of bins
+            t % filters(j) % n_bins = n_words - 1
+
+            ! Allocate and store bins
+            allocate(t % filters(j) % real_bins(n_words))
+            call get_node_array(node_filt, "bins", t % filters(j) % real_bins)
+
+            ! Allow a user to input a lone number which will mean that
+            ! you subivide [0,pi] evenly with the input being the number of bins
+            if (n_words == 1) then
+              Nangle = abs(int(t % filters(j) % real_bins(1)))
+              if (Nangle > 1) then
+                t % filters(j) % n_bins = Nangle
+                dangle = PI / (real(Nangle,8))
+                deallocate(t % filters(j) % real_bins)
+                allocate(t % filters(j) % real_bins(Nangle + 1))
+                do iangle = 1, Nangle + 1
+                  t % filters(j) % real_bins(iangle) = (iangle - 1) * dangle
+                end do
+                t % filters(j) % real_bins(Nangle + 1) = PI
+              else
+                call fatal_error("Number of bins for polar filter must be&
+                     & greater than 1 on tally " // trim(to_str(t % id)) // ".")
+              end if
+
+            end if
+
+          case ('azimuthal')
+            ! Set type of filter
+            t % filters(j) % type = FILTER_AZIMUTHAL
+
+            ! Set number of bins
+            t % filters(j) % n_bins = n_words - 1
+
+            ! Allocate and store bins
+            allocate(t % filters(j) % real_bins(n_words))
+            call get_node_array(node_filt, "bins", t % filters(j) % real_bins)
+
+            ! Allow a user to input a lone number which will mean that
+            ! you subivide [0,2pi] evenly with the input being the number of bins
+            if (n_words == 1) then
+              Nangle = abs(int(t % filters(j) % real_bins(1)))
+              if (Nangle > 1) then
+                t % filters(j) % n_bins = Nangle
+                dangle = TWO * PI / (real(Nangle,8))
+                deallocate(t % filters(j) % real_bins)
+                allocate(t % filters(j) % real_bins(Nangle + 1))
+                do iangle = 1, Nangle + 1
+                  t % filters(j) % real_bins(iangle) = -PI + (iangle - 1) * dangle
```

Comment:
The documentation says 0 to 2pi, but here it is clearly -pi to pi. Is that intentional?